tools/serverpod_cli/test/_test_util/builders/class_definition_builder.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:serverpod_cli/src/analyzer/entities/definitions.dart';
+
+import 'serializable_entity_field_definition_builder.dart';
+
+class ClassDefinitionBuilder {
+  ClassDefinition _classDefinition;
+
+  ClassDefinitionBuilder()
```

Comment:
Suggestion - Build the object once and remove all copyWith methods.

Since we don't have any other use for the copyWith methods so far and there is another solution to build the tests I don't think we should add them yet. Feels like a case of YAGNI.